### ISSUE TITLE
rstudio on windows depends on an OpenSSL version with known vulnerabilities.

### ISSUE THREAD

**Issue by dirks** [rstudio/rstudio#8574]
From https://github.com/rstudio/rstudio/blob/master/dependencies/windows/install-openssl/install-openssl.R

```
URL <- "https://www.openssl.org/source/openssl-1.1.1d.tar.gz"
```

Check https://www.openssl.org/news/vulnerabilities-1.1.1.html for an overview of security vulnerabilities that have been fixed since then.

**Comment by gtritchie** [comment]
Most recent is 1.1.1i from Dec 2020. We are on 1.1.1d from Sept 2019.

**Comment by gtritchie** [comment]
Fixed in v1.4-juliet-rose branch via https://github.com/rstudio/rstudio/pull/8595

**Comment by R-Hannibal** [comment]
Verified Fixed on: RStudio Version - 1.4.1510, R Version - 3.6.3, Windows 10 - Desktop 

Verified that the openssl version has been bumped to 1.1.1g. Not alot to regress outside of that, this is a low risk change that will be best tested via regression and community usage.
